samu.c

```diff
@@ -79,6 +79,59 @@ warnflag(const char *flag)
 		fatal("unknown warning flag '%s'", flag);
 }
 
+void
```

Comment:
This should be `static void` since it is local to this file.

Reply:
yep, sorry, forgot. done.